### PR TITLE
Basic "issues" column

### PR DESCRIPTION
**Pull request by esoergel**
http://manage.dimagi.com/default.asp?147945
@czue

### CODE REVIEW

**Comment by dimagimon** [comment]
There are 7 errors:

* custom/opm/beneficiary.py, line 680 - E129 visually indented line with same indent as next logical line
* custom/opm/tests/test_case_properties.py, line 1 - F401 'datetime' imported but unused
* custom/opm/tests/test_case_properties.py, line 2 - F401 'XFormInstance' imported but unused
* custom/opm/tests/test_case_properties.py, line 3 - F401 'AggressiveDefaultDict' imported but unused
* custom/opm/tests/test_case_properties.py, line 3 - F401 'MockDataProvider' imported but unused
* custom/opm/tests/test_case_properties.py, line 3 - F401 'Report' imported but unused
* custom/opm/tests/test_case_properties.py, line 3 - F401 'offset_date' imported but unused

**Comment by czue** [comment]
looking good to me. i think whatever gets the job done at this point.

**Comment by dimagimon** [comment]
There are 8 errors:

* custom/opm/beneficiary.py, line 678 - E129 visually indented line with same indent as next logical line
* custom/opm/reports.py, line 824 - E226 missing whitespace around arithmetic operator
* custom/opm/tests/test_case_properties.py, line 1 - F401 'datetime' imported but unused
* custom/opm/tests/test_case_properties.py, line 2 - F401 'XFormInstance' imported but unused
* custom/opm/tests/test_case_properties.py, line 3 - F401 'AggressiveDefaultDict' imported but unused
* custom/opm/tests/test_case_properties.py, line 3 - F401 'MockDataProvider' imported but unused
* custom/opm/tests/test_case_properties.py, line 3 - F401 'Report' imported but unused
* custom/opm/tests/test_case_properties.py, line 3 - F401 'offset_date' imported but unused

**Comment by dimagimon** [comment]
There are 7 errors:

* custom/opm/beneficiary.py, line 678 - E129 visually indented line with same indent as next logical line
* custom/opm/tests/test_case_properties.py, line 1 - F401 'datetime' imported but unused
* custom/opm/tests/test_case_properties.py, line 2 - F401 'XFormInstance' imported but unused
* custom/opm/tests/test_case_properties.py, line 3 - F401 'AggressiveDefaultDict' imported but unused
* custom/opm/tests/test_case_properties.py, line 3 - F401 'MockDataProvider' imported but unused
* custom/opm/tests/test_case_properties.py, line 3 - F401 'Report' imported but unused
* custom/opm/tests/test_case_properties.py, line 3 - F401 'offset_date' imported but unused

**Comment by esoergel** [comment]
@czue @sravfeyn updated issues column

File: custom/opm/reports.py [hunk 1]
```diff
@@ -807,6 +807,8 @@ def rows(self):
         return map(self.join_rows, accounts.values())
 
     def join_rows(self, rows):
+        if len(rows) == 1:
+            return rows[0]
```
Comment (esoergel) [inline custom/opm/reports.py]: don't bother zipping if there's only one row

**Pull request merged**